tests/fixtures/synthetic_hf_state.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+#from collections.abc import Iterable
+
+import pytest
```

Comment:
Can the fixtures in this file be moved into the test_sysnthetic_state.py? Would prefer not to add global fixtures that are used by just one test module.

Reply:
we could - though I believe that reverses the feedback from @hanlint ?

Reply:
Ah to clarify, my feedback was to move the test to a `test_...` file so its picked up by pytest. I'm also in favor of moving the fixtures into the `test_..` file as well.